test/MfldSetTac.lean

```diff
@@ -18,55 +18,56 @@ open Lean Meta Elab Tactic
 /-! ## Syntax of objects and lemmas needed for testing `MfldSetTac` -/
 section stub_lemmas
 
-@[mfld_simps] lemma Set.mem_set_of_eq {p : α → Prop} : (x ∈ {y : α | p y}) = p x := sorry
+-- Porting note: this is `Set.mem_setOf_eq` now
+-- @[mfld_simps] lemma Set.mem_set_of_eq {p : α → Prop} : (x ∈ {y : α | p y}) = p x := sorry
```

Comment:
These tests are not mathlib3 tests, instead I added them when the mathlib4 version was written to test it, because at the time `Data.Set.Basic` was not ported.  So you should just delete rather than commenting.